package.json

```diff
@@ -80,7 +84,7 @@
     "rxjs": "^5.1.0",
     "ts-node": "~2.0.0",
     "tslint": "~4.5.0",
-    "typescript": "~2.2.0",
+    "typescript": "~2.3.0",
```

Comment:
I had to update typescript to next minor version because bluebird wasn't compiling.